modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -1,6 +1,6 @@
 = Creating Endpoints
 
-You can create new endpoints from the *Partners* or *Host* page, or by using the Message Flow wizard.
+You can create new endpoints from the *Partners* or *<host>* page, or by using the Message Flow wizard.
```

Comment:
@vijayang Should I also add third-party page to this one?

Reply:
yes!